server/src/models/finances/models.js

```diff
@@ -2,13 +2,88 @@ import _ from 'lodash';
 import mongoose from "mongoose";
 import { 
   number_type,
+  str_type,
   parent_fkey_type,
+  bilingual,
 } from '../model_utils.js';
 import { 
   create_resource_by_foreignkey_attr_dataloader, 
 } from '../loader_utils.js';
 
 export default function(model_singleton){
+  const OrgVoteStatPaSchema = mongoose.Schema({
+    dept_code: parent_fkey_type(),
+    vote_num: str_type,
+    vs_type: number_type,
+    ...bilingual("name", {...str_type, required: true }),
+
+    pa_last_year_5_auth: number_type,
+    pa_last_year_4_auth: number_type,
+    pa_last_year_3_auth: number_type,
+    pa_last_year_2_auth: number_type,
+    pa_last_year_auth: number_type,
+
+    pa_last_year_5_exp: number_type,
+    pa_last_year_4_exp: number_type,
+    pa_last_year_3_exp: number_type,
+    pa_last_year_2_exp: number_type,
+    pa_last_year_exp: number_type,
+  });
+  const OrgVoteStatQfrSchema = mongoose.Schema({
+    dept_code: parent_fkey_type(),
+    vote_num: str_type,
+    vs_type: number_type,
+    ...bilingual("name", {...str_type, required: true }),
+
+    in_year_auth: number_type,
+    in_year_quarter_exp: number_type,
+    in_year_cumul_exp: number_type,
+    last_year_auth: number_type,
+    last_year_quarter_exp: number_type,
+    last_year_cumul_exp: number_type,
+  });
+  const OrgVoteStatEstimatesSchema = mongoose.Schema({
+    dept_code: parent_fkey_type(),
+    vote_num: str_type,
+    vs_type: number_type,
+    ...bilingual("name", {...str_type, required: true }),
+    doc: str_type,
+    
+    est_last_year_4: number_type,
+    est_last_year_3: number_type,
+    est_last_year_2: number_type,
+    est_last_year: number_type,
+    est_in_year: number_type,
+  });
+  const OrgTransferPaymentsSchema = mongoose.Schema({
+    dept_code: parent_fkey_type(),
+    type: str_type,
+    ...bilingual("name", {...str_type, required: true }),
+    pa_last_year_5_auth: number_type,
+    pa_last_year_4_auth: number_type,
+    pa_last_year_3_auth: number_type,
+    pa_last_year_2_auth: number_type,
+    pa_last_year_1_auth: number_type,
+
+    pa_last_year_5_exp: number_type,
+    pa_last_year_4_exp: number_type,
+    pa_last_year_3_exp: number_type,
+    pa_last_year_2_exp: number_type,
+    pa_last_year_1_exp: number_type,
+  });
+  const ProgramSobjsSchema = mongoose.Schema({
+    program_id: parent_fkey_type(),
+    so_num: number_type,
+    pa_last_year_3: number_type,
+    pa_last_year_2: number_type,
+    pa_last_year: number_type,
+  });
+  const ProgramVoteStatSchema = mongoose.Schema({
+    program_id: parent_fkey_type(),
+    pa_last_year_3: number_type,
```

Comment:
This model should have a `vs_type` field too, please add it here/in the schema/in the snapshot query.

Reply:
Oops.. nice catch..!! thank you